### PR TITLE
Resolve external link error in localhost #1183

### LINKED ISSUE #1183: 🐛 [Bug report]: Event link doesn't work on localhost
### Descrcibe the bug?

When you are on localhost, when you click the link to the event page you got access to this link 

http://localhost:3000/https://hustles.reactplay.io/

The cause is the use of <Link></Link> instead of <a> </a>

image

Solution: 
Change the code in headerNav.jsx to get it resolved.

### Steps to reproduce the bug?

Install the application.
Navigate in localhost.
Click on the events button on the header.

### Expected behavior

Be redirected to the https://hustles.reactplay.io/ directly

### Desktop (Please provide your system information)

Windows 11
Vivaldi

### Mobile (Please provide your device information)

_No response_

### Screenshot / Screenshare

image

### Relevant log output

_No response_

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### PR DESCRIPTION
**Pull request by Angra974**
> First thing, PLEASE READ THIS: [ReactPlay Code Review Checklist](https://github.com/reactplay/react-play/wiki/ReactPlay-Code-Review-Checklist)

# Description

When we use the application on localhost, the new events button leads us to a local link instead of an external link.
so http://localhost:3000/https://hustles.reactplay.io
instead of https://hustles.reactplay.io

Fixes #1183 
Change the <Link> used for internal link routing by a <a> link.

## Type of change
Minimal. Change <Link> to <a>

Please delete options that are not relevant.

- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?
No test is needed. Just get the link works as it should. 

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce.

# Checklist:

- [ X] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ X] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

# Screenshots or example output

### CODE REVIEW

**Comment by atapas** [comment]
@Angra974 Fix the lint error plz

image

File: src/common/header/HeaderNav.jsx [hunk 1]
```diff
@@ -88,15 +88,15 @@ const HeaderNav = ({ showBrowse }) => {
             </li>
           )}
           <li>
-            <Link
+            <a
```
Comment (Sachin-chaurasiya) [inline src/common/header/HeaderNav.jsx:91]: @Angra974 , Can you try this?

```suggestion
            <Link
              className="app-header-btn app-header-btn--secondary"
              data-testid="events-btn"
              target="_blank"
              to={pathname:"https://hustles.reactplay.io"}
            >
```

Comment (Angra974) [reply src/common/header/HeaderNav.jsx:91]: Same result.
image

Comment (atapas) [reply src/common/header/HeaderNav.jsx:91]: > @Angra974 , Can you try this?

a should come inside Link, isn't it?

Comment (Sachin-chaurasiya) [reply src/common/header/HeaderNav.jsx:91]: @Angra974 , did you try the same suggestion?

Comment (Angra974) [reply src/common/header/HeaderNav.jsx:91]: > hname:"https://hustles.reactplay.io"}}

Yes, it's ok with your code.

Comment (Angra974) [reply src/common/header/HeaderNav.jsx:91]: I swap my browser to see in another one.

Comment (Angra974) [reply src/common/header/HeaderNav.jsx:91]: No. Didn't work. Try it in Chrome but got the localhost before again. 
image

**Review by Sachin-chaurasiya — APPROVED** [review]

**Review by atapas — APPROVED** [review]
Looks Good

**Pull request merged**